common/config.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"encoding/json"
+	"errors"
+
+	"github.com/coreos/rkt/pkg/config"
+)
+
+const (
+	// configuration directory basenames
+	OldCDB = ""
```

Comment:
const empty string? Probably I am missing some context how does this make sense. Maybe a bit more in the comment why empty string?